components/Footer.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
You can delete this file (and the css module) as we already use the <SocialLinks /> component